assets/js/index.js

```diff
@@ -333,15 +333,18 @@ $(document).ready(function () {
   addScrollHandler();
   // Re-determine on/off of the scrollHanlder when the window is resized
   window.addEventListener("resize", addScrollHandler);
-});
-
-//function to copy link
-function copyLink(item){
+  
+  
+  //function to copy link
+   $(".icon-link").click(function copyLink(){
```

Comment:
This will not work for newly added elements on the example page. Please add it like I mentioned. Also remove the naming of function "copyLink" as it is not necessary.

Don't forget to remove all the "onclick"  attributes in both javascript and HTML files

Reply:
```suggestion
 $("body").on('click',".icon-link",function (){
```